src/benchmarks/micro/Categories.cs

```diff
@@ -31,6 +31,7 @@ public static class Categories
             public const string Virtual = "Virtual";
 
         public const string JSON = "JSON";
+        public const string JsonSerializer = "JsonSerializer";
```

Comment:
I added this to make it easier to run the grouped together tests, locally. If we prefer not to add it, ket me know if there is a better way to do that and we can then remove this category.